### PR TITLE
Add option to add topic to top level

### PR DESCRIPTION
**Pull request by anthonypt87**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "async": "*",
     "prozess": "0.7.0",
     "readable-stream": "1.1.9",
-    "uberclass": "1.0.1"
+    "uberclass": "1.0.1",
+    "zookeeper-uber": "^3.4.6-3-uber-bundled-iostop-yosemite"
```
Comment (Raynos) [inline package.json]: do not add zookeeper back.

Comment (anthonypt87) [inline package.json]: I can't run npm test without it. is this intentional?

Comment (anthonypt87) [inline package.json]: I added it back but I can't get the tests to run without it.

**Comment by anthonypt87** [comment]
ready for you @Raynos

**Comment by Raynos** [comment]
lgtm. Merge when ready.

**Comment by anthonypt87** [comment]
also regarding the bumping versions, will do next time. thanks @Raynos

**Pull request merged**